Gruntfile.js

```diff
@@ -407,7 +407,7 @@ module.exports = function (grunt) {
         exec: {
             localize: 'node scripts/properties2js',
             'localize-dist': 'node scripts/properties2js dist',
-            'clean-nls': 'rm -fr src/nls && git checkout -- src/nls'
+            'clean-nls': 'node_modules/.bin/rimraf src/nls && git checkout -- src/nls'
```

Comment:
I would *strongly* suggest making this an npm script and calling `npm run clean-nls` here, so that there is no guess work about where `rimraf` lives. In NPM scope, it lives "wherever npm has been told to put it", which can't break, whereas this will break if someone's using a custom npm version, or even if npm changes the indexing location again, like it did going from npm2 to npm3.